stacks/go/1.0.2/devfile.yaml

```diff
@@ -1,14 +1,14 @@
 schemaVersion: 2.1.0
 metadata:
-  description: "Go is an open source programming language that makes it easy to build simple, reliable, and efficient software."
+  name: go
   displayName: Go Runtime
+  description: Go 1.16 starter project with a simple HTTP server
```

Comment:
see comment in dotnet50

Reply:
I would say here that we could keep too the original description, like we did in Angular, Python etc. Only an addition of a version (e.g 1.16)